data/human/free worlds 4 epilogue.txt

```diff
@@ -71,9 +71,9 @@ mission "FW Epilogue: Alondo"
 				`	"Sure, I'd be glad to."`
 				`	"Sorry, I'm busy right now."`
 					defer
-			`	You meet up with Alondo at a local pub. He tells you that his life has continued to be quite full since the war ended and the Council was dissolved: he's been placed in charge of the Free Worlds diplomatic corps, whose main job appears to be to make sure all the individual worlds within the coalition feel like their concerns are being listened to. "It means lots of travel," he says, "and lots of fine meals with important people. And they're always interested to hear my stories from the war times."`
+			`	You meet up with Alondo at a local pub. He tells you that his life has continued to be quite full since the war ended and the Council was dissolved: he's been placed in charge of the Free Worlds diplomatic corps, whose main job appears to be to make sure all the individual planets within the government feel like their concerns are being listened to. "It means lots of travel," he says, "and lots of fine meals with important people. And they're always interested to hear my stories from the war times."`
```

Comment:
Is `government` correct here?
And, why not `worlds`?

Reply:
I wanted to avoid using "worlds" since we have "Free Worlds" earlier in the sentence already. I don't see why "government" wouldn't be correct, though. Perhaps something like "nation" might be more accurate, but I've always thought that word to be strange when applied to groups of planets.

Reply:
Honestly may be best to just drop "within the X," it's pretty implicit that "all the planets" applies to the ones represented, not literally all planets everywhere.